src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -71,15 +71,15 @@ public static class NormalizationCatalog
         /// </format>
         /// </example>
         public static LpNormalizingEstimator LpNormalize(this TransformsCatalog catalog, string outputColumnName, string inputColumnName = null,
-            LpNormalizingEstimatorBase.NormFunction normKind = LpNormalizingEstimatorBase.Defaults.Norm, bool ensureZeroMean = LpNormalizingEstimatorBase.Defaults.LpEnsureZeroMean)
-            => new LpNormalizingEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, inputColumnName, normKind, ensureZeroMean);
+            LpNormalizingEstimatorBase.NormFunction norm = LpNormalizingEstimatorBase.Defaults.Norm, bool ensureZeroMean = LpNormalizingEstimatorBase.Defaults.LpEnsureZeroMean)
```

Comment:
>norm [](start = 52, length = 4)

Do you think it make sense to unify names in this one, and one in `TextFeaturizingEstimator` (it's VectorNormalizer)

Reply:
Their ranges are different. `TextFeaturizingEstimator` contains `None` but `LpNormalizer` doesn't, but it doesn't make sense to add `None` to the enum of `LpNormalizer`.

---
In reply to: [264822764](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264822764) [](ancestors = 264822764)

Reply:
just in case, I'm not taking of type of this parameter, I'm talking about it's name. 

---
In reply to: [264872101](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264872101) [](ancestors = 264872101,264822764)

Reply:
I will use `norm` in `TextFeaturizingEstimator`.

---
In reply to: [264873318](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264873318) [](ancestors = 264873318,264872101,264822764)